Calculator/README.md

```diff
@@ -0,0 +1,29 @@
+# Calculator Instructions:
+
+## Buttons.
+
+-Each button has assigned it's respective symbol it uses, simply press the button with the desired symbol and it will appear on the display of the calculator.
+
+### Special Characters.
+
+-The button with the flask is the calculators scientific mode to use more difficult equations and problems.
```

Comment:
this was (still is, in fact) a great idea, but its not implemented (yet)
How about updating the docs to reflect the current development state? Maybe putting a line about it being still in development, or will be available only in the React version of this calculator, whichever seems best for you will be fine. What do you think about it?